src/test/groovy/org/sonarqube/gradle/FunctionalTests.groovy

```diff
@@ -105,6 +106,7 @@ class FunctionalTests extends Specification {
         def props = new Properties()
         props.load(outFile.newDataInputStream())
         props."sonar.java.jdkHome".contains('8')
+        new File(props."sonar.java.jdkHome").exists()
         props."sonar.java.source" == '8'
         props."sonar.java.target" == '8'
     }
```

Comment:
I feel that the next test `"set jdkHome, source and target for 'java' projects from task release"` might be missing the same test is missing the test about the actual presence of the property.

Also, what about adding a test targeting misconfiguration? or jdk not found?